Unix/configure

```diff
@@ -2298,6 +2300,12 @@ then
     cp $rootrelative/etc/omigen.conf \$destdir/$sysconfdir/omigen.conf
 fi
 
+
+if test ! -f "\$destdir/$sysconfdir/omireg.conf"
+then 
+    cp $rootrelative/etc/omireg.conf \$destdir/$sysconfdir/omireg.conf
+fi
+
```

Comment:
All of the edits in configure are cut-n-paste copies of how omicli.conf, omigen.conf, and omiserver.conf are handled.  The behavior of the configuration is the same for every other conf file.